components/steps/Status.tsx

```diff
@@ -17,7 +17,7 @@ const getStatusLabel = (profile: Profile): string => {
     return "Unverified";
   }
   if (status === "checkedIn") return "Checked In";
-  if (status === "verified" && submittedAt !== null) {
```

Comment:
I hate how JavaScript has `undefined` and `null`... Because `submittedAt` was NULL in the DB, Sequelize didn't return it so technically submittedAt was `undefined`. Because we explicitly check for null, we were getting some logic errors (showing users that they submitted an app when they just created an account) 

Slight yikes